src/compiler/Stage/Emit/JavaScript.elm

```diff
@@ -45,6 +45,9 @@ emitExpr located =
         Plus e1 e2 ->
             "(" ++ emitExpr e1 ++ " + " ++ emitExpr e2 ++ ")"
 
+        Cons e1 e2 ->
+            "[].concat([" ++ emitExpr e1 ++ "]," ++ emitExpr e2 ++ ")"
```

Comment:
I know it's inconsequential, but it just dawned on me that it can be `[e1].concat(e2)` instead of `[].concat([e1],e2)`. WDYT?

Reply:
Good idea, I've implemented this 👍